plasmapy/constants/atomic.py

```diff
@@ -687,11 +709,13 @@ def ion_mass(argument, Z=1, mass_numb=None):
     Parameters
     ----------
     argument: string or integer
-        A string representing an element or isotope, or an integer representing
-        the atomic number of an element.
+        Returns the mass of an ion by finding the standard atomic
+        weight of an element or the atomic mass of an isotope, and then
+        accounting for the change in mass due to loss of electrons from
+        ionization.
```

Comment:
Thanks for finding this!  One of the things I still need to do is to go through the docstrings and make sure everything they are consistent with what the functions are actually doing, so this is very helpful.